ci-operator/config/openshift-kni/mixed-cpu-node-plugin/openshift-kni-mixed-cpu-node-plugin-main.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  os-min8:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+binary_build_commands: make fmt vet build-plugin
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tools
```

Comment:
we should probably stop using this system, inherited ages ago from earliest PAO and never really re-evaluated. Not blocking for this PR though.